### PR TITLE
Clean up error messages in Productiv

### LINKED ISSUE #224: Fix all error messages for invalid input commands
-grammar
-punctuation
-only compulsory fields should include "should not be left blank" (add)
-remove unnecessary parts ("can accept any value", "single digit days, months,...")

### PR DESCRIPTION
**Pull request by gabztcr**
Fix #224 

-Did not remove error message for "Description" and "Location" fields yet. Waiting for Wenjie to amend code first to allow blank fields to pass for optional fields, before deleting these two fields' error messages (they will then accept everything including blanks)

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -255,12 +255,12 @@ Adds a meeting to your meeting list.
...
-* `FROM` is the start date time of the meeting in DD-MM-YYYY HH:mm format.
-* `TO` is the end date time of the meeting in DD-MM-YYYY HH:mm format.
-* `FROM` and `TO` can be in the past but must not be earlier than the year 2019.
+* `FROM` is the start date time of the meeting in dd-MM-yyyy HH:mm format.
+* `FROM` can accept dates in the past but must not be earlier than the year 2019.
+* `TO` is the end time of the meeting in HH:mm format.
 * `DESCRIPTION` contains additional details about the meeting, e.g. agenda.
-* `CONTACTS` represents the contacts involved in the meeting. 
-* `CONTACTS` is a comma-separated string of names.
+* `CONTACTS` represents the contact(s) involved in the meeting.
```
Comment (MerlinLim) [inline docs/UserGuide.md:262]: Include optional plural for contacts in line 272

File: src/main/java/seedu/address/model/util/Time.java [hunk 2]
```diff
@@ -8,8 +8,7 @@
 
 public class Time implements Comparable<Time> {
     public static final String TIME_REGEX = "(([0-1]\\d)|(2[0-3])):([0-5]\\d)";
-    public static final String CONSTRAINTS = "%s should be in the format of HH:mm, "
-            + "hour and minute must start with a leading zero.";
+    public static final String CONSTRAINTS = "%s should be in HH:mm format. ";
```
Comment (MerlinLim) [inline src/main/java/seedu/address/model/util/Time.java:11]: Remove additional white space at the end

File: src/main/java/seedu/address/model/util/Description.java [hunk 3]
```diff
@@ -13,6 +13,7 @@
 public class Description {
 
     public static final String EMPTY_DESCRIPTION_FIELD = "-";
+    // TODO: remove after allowing blank optional fields to pass.
     public static final String MESSAGE_CONSTRAINTS = "Descriptions can take any values, and it should not be blank";
```
Comment (MerlinLim) [inline src/main/java/seedu/address/model/util/Description.java:17]: missing period for constriants

File: src/main/java/seedu/address/model/meeting/meeting/Location.java [hunk 4]
```diff
@@ -11,6 +11,7 @@
  */
 public class Location {
     public static final String EMPTY_LOCATION_FIELD = "-";
+    // TODO: remove after allowing blank optional fields to pass.
     public static final String MESSAGE_CONSTRAINTS = "Locations can take any values, and it should not be blank";
```
Comment (MerlinLim) [inline src/main/java/seedu/address/model/meeting/meeting/Location.java:15]: Missing period

**Review by MerlinLim — COMMENTED** [review]
Minor changes, other than that LGTM.

File: src/main/java/seedu/address/model/meeting/meeting/Location.java (hunk 4 above)
Comment (gabztcr) [reply src/main/java/seedu/address/model/meeting/meeting/Location.java:15]: Message will be removed later on, so will not be making extra changes.

File: src/main/java/seedu/address/model/util/Description.java (hunk 3 above)
Comment (gabztcr) [reply src/main/java/seedu/address/model/util/Description.java:17]: Message will be removed later on, so will not be making extra changes.

File: docs/UserGuide.md (hunk 1 above)
Comment (gabztcr) [reply docs/UserGuide.md:261]: Will amend to "`CONTACTS` has no relation to any contact in your contact list" to account for both

Comment (gabztcr) [reply docs/UserGuide.md:261]: Reworded to `As `CONTACTS` is not related to your contact list, you can include those not present in it.`

**Review by MerlinLim — APPROVED** [review]
Synchronise schedule order for Deliverable card and deliverable mode list panel. Other than that LGTM

**Pull request merged**